web/src/containers/ApdApplication.js

```diff
@@ -89,20 +98,16 @@ class ApdApplication extends Component {
 ApdApplication.propTypes = {
   apdSelected: PropTypes.bool.isRequired,
   dirty: PropTypes.bool.isRequired,
+  isAdmin: PropTypes.bool.isRequired,
   place: PropTypes.object.isRequired,
   selectApdOnLoad: PropTypes.func.isRequired
 };
 
-const mapStateToProps = ({
-  apd: { data },
-  dirty: { dirty },
-  user: {
-    data: { state }
-  }
-}) => ({
-  apdSelected: !!data.id,
-  dirty,
-  place: state
+const mapStateToProps = state => ({
+  apdSelected: getIsAnAPDSelected(state),
+  dirty: getIsDirty(state),
+  isAdmin: getIsAdmin(state),
+  place: getUserStateOrTerritory(state)
```

Comment:
Converted all of these to selectors because it was cleaner.  A little piece at a time...